snmp/datadog_checks/snmp/data/profiles/idrac.yaml

```diff
@@ -0,0 +1,48 @@
+# Profile for iDRAC devices
+sysobjectid: 1.3.6.1.4.1.674.10892.*
+metrics:
+  - MIB: IDRAC-MIB-SMIv2
+    table: systemStateTable
+    symbols:
+      - systemStateChassisStatus
+      - systemStatePowerUnitStatusRedundancy
+      - systemStatePowerSupplyStatusCombined
+      - systemStateAmperageStatusCombined
+      - systemStateCoolingUnitStatusRedundancy
+      - systemStateCoolingDeviceStatusCombined
+      - systemStateTemperatureStatusCombined
+      - systemStateMemoryDeviceStatusCombined
+      - systemStateChassisIntrusionStatusCombined
+      - systemStatePowerUnitStatusCombined
+      - systemStateCoolingUnitStatusCombined
+      - systemStateProcessorDeviceStatusCombined
+      - systemStateTemperatureStatisticsStatusCombined
+    metric_tags:
+      - tag: chassis_index
+        column: systemStatechassisIndex
+  - MIB: IDRAC-MIB-SMIv2
+    table: enclosurePowerSupplyTable
+    symbols:
+      - enclosurePowerSupplyState
+    metric_tags:
+      - tag: supply_name
+        column: enclosurePowerSupplyName
+  - MIB: INTEL-LAN-ADAPTERS-MIB
+    table: genericAdaptersTrafficStatsAttrTable
+    forced_type: monotonic_count
+    symbols:
+      - adapterRxPackets
+      - adapterTxPackets
+      - adapterRxBytes
+      - adapterTxBytes
+      - adapterRxErrors
+      - adapterTxErrors
+      - adapterRxDropped
+      - adapterTxDropped
+      - adapterRxMulticast
+      - adapterCollisions
+    metric_tags:
+      - tag: adapter
+        column: adapterName
+        MIB: INTEL-LAN-ADAPTERS-MIB
```

Comment:
What are those two lines? Do we miss a part of the profile?

Reply:
Which 2 lines? MIB/table? I think I can remove the MIB one, it's probably redundant.

Reply:
I meant that we don't seem to do anything with those tags: https://github.com/DataDog/integrations-core/blob/cb0a105c940a4d939f7e0d1b8e31a2f020c7d6be/snmp/datadog_checks/snmp/snmp.py#L475-L490

Reply:
Oh I didn't know you could use a custom MIB/table in metric_tags and I had a hard time finding this with the code.

Nice 👍